tests/unit/test_viz.py

```diff
@@ -44,3 +47,39 @@ def test_to_pi_string():
 
     # Test for tex=False
     assert to_pi_string(-np.sqrt(np.pi) / 2, tex=False) == "-\\sqrt{\\pi}/2"
+
+
+def test_draw_EGraph_Bell():
+    """Test for the draw method of EGraph of Bell state."""
+    # Bell state EGraph
+    edge = [(0, 0, 0), (0, 0, 1)]
+    bell_state = EGraph()
+    bell_state.add_edge(*edge, color="MidnightBlue")
+
+    # Test for drawing the EGraph
+    f, a = draw_EGraph(bell_state)
+    plt.close()
+
+    assert len(a.get_xticks()) == 1
+    assert a.get_xlim() == (-1, 1)
```

Comment:
Are there a few other small things we can test here? One thing could be loyalty to the plot params (see Example usage and tests here: https://github.com/XanaduAI/flamingpy/pull/20)

Reply:
I would prefer to put that in another PR if that's cool with you (and maybe open an issue for this now)

Reply:
Sounds good!